template.md

```diff
@@ -10,6 +10,20 @@
 - **Location**: CITY, COUNTRY
 - **Attendee information**: LINK TO REFLECTOR
 
+```mermaid
+---
+displayMode: compact
+---
+gantt
+    dateFormat  YYYY MM DD HH mm Z
```

Comment:
```suggestion
    title TBDth meeting
    dateFormat  YYYY MM DD HH mm Z
```

Reply:
I found this unnecessary (because it is listed at the top of the readme). I omit the title to reduce the maintenance cost of the agenda editor.